images/linux/scripts/installers/azpowershell.sh

```diff
@@ -17,7 +17,8 @@ fi
 
 # Install Azure CLI (instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 for version in ${versions[@]}; do
-    pwsh -Command "Save-Module -Name Az -LiteralPath /usr/share/az_$version -RequiredVersion $version -Force"
+    tlsCommand="[Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12"
```

Comment:
We don't need to set up SecurityProtocol, because it's already set to Tls123:

PS > ${function:Save-Module}:

```
 Begin {
                      # Change security protocol to TLS 1.2
                      $script:securityProtocol = [Net.ServicePointManager]::SecurityProtocol
                      [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

                      Install-NuGetClientBinaries -CallerPSCmdlet $PSCmdlet -Proxy $Proxy -ProxyCredential
              $ProxyCredential

                      # Module names already tried in the current pipeline for InputObject parameterset
                      $moduleNamesInPipeline = @()
 }
```

Reply:
yes, true, find out that while save-module is executing, tls if forced to 1.2 and is set to default at the end of execution